content/install_maintain/installation/installing-external-db-and-queue-cluster.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: bt_wiki
+title: Installing and Configuring External DB And Rabbitmq Within Distributed Cluster
+description: Install and Configure External DB And Rabbitmq within Distributed Cluster . 
+category: Installation
+draft: false
+weight: 5
+---
+
+
+# Installing and Configuring External DB And Rabbitmq Within Distributed Cluster
+
+When installing the Cloudify cluster, the user can use externally PostgreSQL database and RabbitMQ.
+This page is a guide for installing such services.
+
+### PostgreSQL Database Cluster
+
+The PostgreSQL database high-availability cluster is comprised of 3 nodes (Cloudify best-practice) or more.
+
+**Note** Make sure the following ports are open for each node:
+
+ Port      | Description
+-----------|------------
+ tcp/2379  | etcd port.
+ tcp/2380  | etcd port.
+ tcp/5432  | PostgreSQL connection port.
+ tcp/8008  | Patroni control port.
```

Comment:
This one is not external DB, it doesn't match the title. We need either to move it to a different page or adjust the page title

Reply:
Well  i think that the content is good , the ports to open are the same i guess @geokala ?
If it is . so i will change the title. notice that next we have this title.
#### Externally Hosted PostgreSQL Database Installation

Reply:
Removed unnecessary parts.